release-notes/elastic-cloud-serverless/index.md

```diff
@@ -9,6 +9,67 @@ products:
 # {{serverless-full}} changelog [elastic-cloud-serverless-changelog]
 Review the changes, fixes, and more to {{serverless-full}}.
 
+## November 10, 2025 [serverless-changelog-11102025]
+
+### Features and enhancements [serverless-changelog-11102025-features-enhancements]
+
+* Enables {{ml}} to manage anomaly detection results indices in Elasticsearch Serverless [#136065]({{es-pull}}136065)
```

Comment:
The "in Elasticsearch Serverless" IMO implies that this only applies to the Elasticsearch project, which seems unlikely. I think the original summary from the changelog is better:
```suggestion
* Nightly maintenance for anomaly detection results indices to keep to manageable size [#136065]({{es-pull}}136065)
```